src/bindgen/ir/constant.rs

```diff
@@ -252,6 +274,12 @@ impl Literal {
                 right.write(config, out);
                 write!(out, ")");
             }
+            Literal::Cast { ref ty, ref value } => {
```

Comment:
We should probably wrap the whole expression in parens.

Reply:
Casts have pretty high precedence (https://en.cppreference.com/w/c/language/operator_precedence), so I don't think that's necessary. I can add it if you really want though.

Reply:
Macros that expand to complex stuff without parens always make me a bit nervous, but you're probably right, we can always change this if needed.